huff_codegen/src/lib.rs

```diff
@@ -135,13 +137,16 @@ impl Codegen {
 
         res.utilized_tables.iter().try_for_each(|jt| {
             table_offsets.insert(jt.name.to_string(), table_offset);
-            let size = match bytes32_to_string(&jt.size, false).as_str().parse::<usize>() {
+            let size = match usize::from_str_radix(bytes32_to_string(&jt.size, false).as_str(), 16) {
```

Comment:
Maybe use your new `hex_to_usize` helper here?

Reply:
ye defined that after :P will do now